apps/ledger-live-desktop/src/renderer/drawers/Drawer.tsx

```diff
@@ -18,23 +18,23 @@ const transitionStyles = {
   },
 };
 const DURATION = 200;
-const Bar = styled.div.attrs<{ state: TransitionStatus }>(props => ({
+const Bar = styled.div.attrs<{ state: TransitionStatus; padding: boolean }>(props => ({
```

Comment:
just a suggestion, it's up to you: maybe `withPadding` would be a tiny bit clearer (with `padding` I would expect a `number` value)